my_dagster_project/repository.py

```diff
@@ -1,10 +1,47 @@
-from dagster import load_assets_from_package_module, repository, define_asset_job
+import os
+
+from dagster import define_asset_job, load_assets_from_package_module, repository, with_resources
+from dagster_aws.s3 import s3_pickle_io_manager, s3_resource
 
 from my_dagster_project import assets
 
 
 @repository
 def my_dagster_project():
+
+    # # To take advantage of Dagster's incremental re-execution functionality (e.g. retry from failure),
+    # # you'll need to set up an IO manager that can move the data across runs.
+    # # Learn more at https://docs.dagster.io/concepts/io-management/io-managers#applying-io-managers-to-assets
+    # # NOTE: Uncomment the example code below to use an s3_pickle_io_manager to pass data between runs.
```

Comment:
A little confusing here that there's no code directly below this line.

Reply:
reformatted the comment - hope this is more clear